packages/amplication-server/src/core/prismaSchemaUtils/schema-utils.spec.ts

```diff
@@ -44,8 +44,17 @@ describe("schema-utils", () => {
   describe("formatFieldName", () => {
     it("should format the field name in camelCase", () => {
       (isReservedName as jest.Mock).mockReturnValueOnce(false);
-      const result = formatFieldName("test_field");
-      expect(result).toEqual("testField");
+      expect(formatFieldName("Test_5_Rank_X_out_of_Y")).toEqual(
+        "test5RankXOutOfY"
+      );
+      expect(formatFieldName("ABC_Review_Form_num7_Comments")).toEqual(
+        "abcReviewFormNum7Comments"
+      );
+      expect(formatFieldName("Another_test_string")).toEqual(
+        "anotherTestString"
+      );
+      expect(formatFieldName("test")).toEqual("test");
+      expect(formatFieldName("TEST_CASE")).toEqual("testCase");
```

Comment:
you could use test cases instead of a lot of expectations. This will make it easier to understand the failing case

https://jestjs.io/docs/api#testeachtablename-fn-timeout